travelpal/lib/travelpal_web/controllers/flight_controller.ex

```diff
@@ -1,6 +1,14 @@
 defmodule TravelpalWeb.FlightController do
   use TravelpalWeb, :controller
 
+  # @TODO decide if other functions are needed
+  def flight_url, do: "https://api.skypicker.com/flights"
```

Comment:
`flight_url()` and `get_flights_to_from()` are defined twice. I'm guessing this was just a merge error.